src/Core/ClassInfo.php

```diff
@@ -20,14 +22,6 @@
  */
 class ClassInfo
```

Comment:
Please implement `Flushable` as well so that persistent cache is flushed appropriately.